src/components/App/App.component.jsx

```diff
@@ -1,16 +1,105 @@
 import React, { useLayoutEffect } from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
+import styled from 'styled-components';
+import breakpoint from '../../common/breakpoint';
 import AuthProvider from '../../providers/Auth';
 import HomePage from '../../pages/Home';
 import LoginPage from '../../pages/Login';
 import NotFound from '../../pages/NotFound';
 import SecretPage from '../../pages/Secret';
 import Private from '../Private';
-import Fortune from '../Fortune';
+// import Fortune from '../Fortune';
+import Hamburger from '../Hamburger';
+import User from '../User';
+import SearchIcon from '../SearchIcon';
 import Layout from '../Layout';
 import { random } from '../../utils/fns';
 
+const Header = styled.nav`
```

Comment:
All this styling can be extracted to a file, for example, `App.styled.js` to separate styling from component logic, also I see that `Header` is the only styled component, the following classes can be converted to styled-components to avoid mixing styling approaches

Reply:
Done, exported all the styling to `App.styled.js`